platform/security/handler/security-handler-pki/src/test/java/org/codice/ddf/security/handler/pki/CrlCheckerTest.java

```diff
@@ -119,6 +121,48 @@ public void testGetPropertiesFailsBothCertsPass() throws CertificateException {
         assertThat(crlChecker.passesCrlCheck(certs), equalTo(true));
     }
 
+    @Test
+    public void testEnabledCrlFromURLCertFails() throws CertificateException {
+
+        CrlChecker crlChecker = getConfiguredCrlChecker("encryption-crl-none.properties");
+
+        //Get a URL path for the file, this will generate a URL
+        String urlPath = ClassLoader.getSystemResource("crl-revoked.pem")
+                .toString();
+        crlChecker.setCrlLocation(urlPath);
+
+        // First cert should fail
+        String certificateString = getRevokedCert();
+        X509Certificate[] certs = extractX509CertsFromString(certificateString);
+        assertThat(crlChecker.passesCrlCheck(certs), equalTo(false));
+
+        // Second cert should pass
+        certificateString = getUnrevokedCert();
```

Comment:
Hmm, well the idea is they're both being checked against the input CRL to make sure the CRL was properly processed, making sure unrevoked certs pass and revoked ones don't. I don't see how splitting them could hurt though.